src/Nav.js

```diff
@@ -113,24 +116,25 @@ Nav.propTypes = {
    * HTML id for the inner `ul` element
    */
   ulId: React.PropTypes.string,
-  expanded: React.PropTypes.bool,
   navbar: React.PropTypes.bool,
   eventKey: React.PropTypes.any,
   pullRight: React.PropTypes.bool,
-  right: React.PropTypes.bool
+  right: deprecated(React.PropTypes.bool, 'Use the `pullRight` prop instead')
```

Comment:
A lot of the navbar examples do explicitly add `navbar-right|left` to the sub components, of which Nav is sort of in a middle space there. I figure since other components (dropdown) have `pullRight` why not just go with that and prefix it correctly in the navbar context.